tests/test_gcvit.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+
+
+import tensorflow as tf
+from gcvit import GCViTTiny
+from skimage.data import chelsea
+
+
+class TestGCViT(unittest.TestCase):
+    def test_prediction(self):
+        model = GCViTTiny(pretrain=True)
+        img = tf.keras.applications.imagenet_utils.preprocess_input(chelsea(), mode='torch') # Chelsea the cat
```

Comment:
It looks like our tests run offline, so this is failing trying to pull it down.

Is it possible to pre-pull anything you need into the test folder?

Reply:
You can ignore the prophet test failure though, I'm fixing that in the main build.